api/feature.go

```diff
@@ -47,6 +47,19 @@ type Feature struct {
 
 	// Variants is a list of variants of the feature toggle.
 	Variants []VariantInternal `json:"variants"`
+
+	// Dependencies is a list of feature toggle dependency objects
+	Dependencies *[]FeatureDependencies `json:"dependencies"`
+}
+
+type FeatureDependencies struct {
```

Comment:
I called this just Dependency in Node SDK with an option to other dependencies. The property inside Feature is indicating what type of dependency it is. If we don't expose this type in the public API then I'm ok with any name.

Reply:
Dependency works fine